build/config.props

```diff
@@ -36,15 +36,17 @@
 
   <!-- Nuspec defaults -->
   <PropertyGroup>
-    <Authors>NuGet</Authors>
-    <ProjectUrl>https://github.com/NuGet/NuGet.Client</ProjectUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</PackageLicenseUrl>
+    <Authors>Microsoft</Authors>
+    <ProjectUrl>https://aka.ms/nugetprj</ProjectUrl>
+    <PackageLicenseUrl>https://github.com/NuGet/NuGet.Client/blob/dev/LICENSE.txt</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```

Comment:
Should the package license url point to a license.txt with a commit hash rather than dev?

Cause this can change from the time the package was built.